File: FaceSample/app/src/main/java/com/regula/facesample/items/other/LocalizationItem.java
package com.regula.facesample.items.other;

import android.content.Context;

import com.regula.facesample.items.CategoryItem;
import com.regula.facesdk.FaceSDK;

/**
 * Created by Sergey Yakimchik on 24.08.21.
 * Copyright (c) 2021 Regula. All rights reserved.
 */

public class LocalizationItem extends CategoryItem {

    @Override
    public void onItemSelected(Context context) {
        // uncomment in res/values/strings.xml:
        // hint.fit resource


        //This is another approach of customizing localization by overriding strings in callback
        FaceSDK.Instance().setLocalizationCallback(stringId -> {
            if(stringId.equals("livenessGuide.head"))
                return "this is custom selfie time string string";
            return null;
        });

        FaceSDK.Instance().startLiveness(context, livenessResponse -> { });
    }

    @Override
    public String getTitle() {
        return "Custom localization";
    }

    @Override
    public String getDescription() {
        return "Localization hook example.";
    }
}

File: BasicSample/app/src/main/java/com/regula/facesamplekotlin/items/other/LocalizationItem.kt
package com.regula.facesamplekotlin.items.other

import android.content.Context
import com.regula.facesamplekotlin.category.CategoryItem
import com.regula.facesdk.FaceSDK
import com.regula.facesdk.model.results.LivenessResponse

class LocalizationItem : CategoryItem() {

    override fun onItemSelected(context: Context) {
        // uncomment in res/values/strings.xml:
        // hint.fit resource

        //This is another approach of customizing localization by overriding strings in callback

        // uncomment in res/values/strings.xml:
        // hint.fit resource


        //This is another approach of customizing localization by overriding strings in callback
        FaceSDK.Instance().setLocalizationCallback { stringId ->
            if (stringId.equals("livenessGuide.head")) return@setLocalizationCallback "this is custom selfie time string string"
            null
        }

        FaceSDK.Instance().startLiveness(context) { livenessResponse: LivenessResponse? -> }
    }

    override val title: String
        get() =  "Custom localization"

    override val description: String
        get() =  "Localization hook example."
}